demo.html

```diff
@@ -4,21 +4,174 @@
 <head>
   <meta charset="utf-8">
   <title>Google Analytics Polymer Elements Demo</title>
-  <script src="../webcomponentsjs/webcomponents.js"></script>
+  <script src="../webcomponentsjs/webcomponents-lite.js"></script>
   <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:700,400,300">
-  <link rel="stylesheet" href="demo.css" shim-shadowdom>
   <link rel="import" href="../google-signin/google-signin.html">
   <link rel="import" href="google-analytics.html">
+
 </head>
-<body unresolved>
+<body >
+<style is="x-style">
+    body {
+      background: #eee;
+      color: #222;
+      font: 13px/1.3 'Open Sans', sans-serif;
+      margin: 0;
+      padding: 0;
+    }
+
+    header {
+      background-color: #333;
+      box-sizing: border-box;
+      color: #eee;
+      display: table;
+      padding: 1.2em 1.5em;
+      width: 100%;
+    }
+    header > h1 {
+      display: table-cell;
+      font-size: 1.667em;
+      font-weight: 400;
+      margin: 0;
+      vertical-align: middle;
+      width: 100%;
+    }
+    header > h1 em {
+      font-weight: 300;
+      font-size: .8em;
+      font-style: normal;
+      margin-left: .5em;
+      opacity: .6;
+    }
+    header > google-signin {
+      display: table-cell;
+      vertical-align: middle;
+      white-space: nowrap;
+    }
+
+    /**
+     * .control styles.
+     */
+     google-analytics-view-selector::shadow .control,
+     google-analytics-date-selector::shadow .control {
+      display: inline-block;
+      margin: 0 1em 1.5em 0;
+    }
+    google-analytics-view-selector::shadow .control:last-child,
+    google-analytics-date-selector::shadow .control:last-child {
+      margin-right: 0;
+    }
+    google-analytics-view-selector::shadow .control > label,
+    google-analytics-date-selector::shadow .control > label {
+      display: block;
+      padding: 0 0 .4em 2px;
+    }
+
+    /**
+     * Common styles.
+     */
+    google-analytics-view-selector,
+    google-analytics-date-selector {
+      border: 1px solid #ccc;
+      border-radius: 4px;
+      background: #fff;
+      float: left;
+      margin: 0 1.5em 1.5em 0;
+      padding: 1.5em 1.5em 0;
+    }
+    google-analytics-chart {
+      border: 1px solid #ccc;
+      border-radius: 4px;
+      background: #fff;
+      float: left;
+      margin: 0 1.5em 1.5em 0;
+      padding: 1.5em;
+      position: relative;
+      transition: opacity .2s ease;
+    }
+    google-analytics-chart:not([rendered]) {
+      display: none;
+    }
+
+    google-analytics-chart:first-of-type {
+      clear: both;
+    }
+    google-analytics-chart h3 {
+      border: 0;
+      font-size: 1.3em;
+      font-weight: 300;
+      margin: 0;
+      padding: 0;
+    }
 
+
+    /**
+     * <google-analytics-view-selector> styles.
```

Comment:
This should be `<google-analytics-dashboard>` (I know, it was already like that, but may as well fix anyway).